__tests__/integration.test.js

```diff
@@ -188,4 +196,100 @@ describe('POST /api/articles/:article_id/comments', () => {
             expect(body.msg).toBe("Username Not Found")
         })
     });
+    test('status 404: returns no article found for article_id when passed an article_id which does not exist', () => {
+        const comment = {
+            username: "lurker",
+            body: "new comment"
+        }
+        return request(app)
+        .post("/api/articles/999999999/comments")
+        .send(comment)
+        .expect(404)
+        .then(({body}) => {
+            expect(body.msg).toBe("No article found for article_id: 999999999")
+        })
+    });
+    test('status 400: returns Bad Request when passed an invalid id (not a number)', () => {
+        const comment = {
+            username: "lurker",
+            body: "new comment"
+        }
+        return request(app)
+        .post("/api/articles/notAnId/comments")
+        .send(comment)
+        .expect(400)
+        .then(({body}) => {
+            expect(body.msg).toBe("Bad Request")
+        })
+    });
+});
+describe('PATCH /api/articles/:article_id', () => {
+    test('status 200: should update an article (given article_id) votes property modified by the inc_votes value provided within the request body, and return the updated article', () => {
+        const newVote = {
+            inc_votes: -20
+        }
+        return request(app)
+        .patch("/api/articles/1")
+        .send(newVote)
+        .expect(200)
+        .then(({body}) => {
+            expect(body.article).toMatchObject({
+                article_id: 1,
+                title: "Living in the shadow of a great man",
+                topic: "mitch",
+                author: "butter_bridge",
+                body: "I find this existence challenging",
+                created_at: "2020-07-09T20:11:00.000Z",
+                votes: 80,
+                article_img_url:
+                "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+            })
+        })
+    });
```

Comment:
Could be worth adding a 200 test where inc_votes is positive instead

Reply:
i've added this, thanks so much 😄